tools/bin/cassandra.in.sh

```diff
@@ -44,77 +54,70 @@ cassandra_storagedir="$SCYLLA_HOME/data"
 # JAVA_HOME can optionally be set here
 #JAVA_HOME=/usr/local/jdk6
 
-
 CONFIG_FILE_REALPATH=$(realpath "$SCYLLA_CONF/cassandra.yaml")
 # If scylla.yaml is found - use it as the config file.
 if [ -f "$SCYLLA_CONF/scylla.yaml" ]; then
-    CONFIG_FILE_REALPATH=$(realpath "$SCYLLA_CONF/scylla.yaml")
+  CONFIG_FILE_REALPATH=$(realpath "$SCYLLA_CONF/scylla.yaml")
 fi
 
 CONFIGURATION_FILE_OPT="-Dcassandra.config=file://$CONFIG_FILE_REALPATH"
 
-for jar in "$SCYLLA_HOME"/tools/lib/*.jar; do
-    CLASSPATH="$CLASSPATH:$jar"
-done
-
-for jar in "$SCYLLA_HOME"/lib/*.jar; do
-    CLASSPATH="$CLASSPATH:$jar"
-done
+CLASSPATH="${CLASSPATH}:$(find "$SCYLLA_HOME/lib" "$SCYLLA_HOME/build/lib" "$SCYLLA_HOME/tools" -name '*.jar' -print 2>/dev/null | paste -sd ":" -)"
+CLASSPATH="${CLASSPATH#:}"
```

Comment:
Albeit odd this is to allow running from ./build, ./tools and docker

> gh repo clone daniel-boll/cassandra-stress -- --depth 1 --branch fix/scripts-path
> cd cassandra-stress
> ant -Drelease=true artifacts
> docker buildx build --build-arg='CASSANDRA_STRESS_VERSION=3.12.0' -t scylla-daniel:cassandra-stress .

Now with that the three ways

> ./tools/bin/cassandra-stress
> ./build/dist/tools/bin/cassandra-stress
> docker run scylla-daniel:cassandra-stress